### DISCUSSION TITLE
What is the maximum JSON file size that can be parsed?

### DISCUSSION THREAD

**Discussion by mementoseeds** [nlohmann/json#2833] [Q&A] [answered]
As the title suggets, is there an upper limit for the file size that can be parsed? For example if I whip out a 200 MB or a 1 GB json file, could it be parsed as long as the computer has enough memory?

**Comment by nlohmann** [comment, marked as answer]
Yes, the only limit is your memory. Note that the library should be compiled in release mode to avoid a too big overhead.
